### DISCUSSION TITLE
Beatmap overlay leaderboard(s) don't load

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16902] [Q&A] [answered]
I don't have supporter so i can't confirm for the other 2 leaderboards there. Global one loads up fine on previous release.

![image](https://user-images.githubusercontent.com/53081750/154471082-4434e9cc-3a97-4f0f-93b0-b6d319c7d985.png)
[network.log](https://github.com/ppy/osu/files/8088184/network.log)
[runtime.log](https://github.com/ppy/osu/files/8088185/runtime.log)
[database.log](https://github.com/ppy/osu/files/8088186/database.log)

**Comment by smoogipoo** [comment, marked as answer]
Will be fixed by https://github.com/ppy/osu/pull/16903
